helloworld.java

```diff
@@ -1,5 +1,5 @@
 public class helloworld {
     public static void main(String[] args) {
-        System.out.println("Hello World!");
+        System.out.println("ALL CAPS!");
```

Comment:
I meant hello world in all caps 😢 
"HELLO WORLD!"

Reply:
I know:)

Reply:
![image](https://github.com/tim61114/CS690_Class_lab1/assets/11361069/86d55a6d-5b6a-4ed1-b931-d0783490a2b6)